src/darwin/policy-softener-glue.c

```diff
@@ -33,4 +34,42 @@ _frida_electra_policy_softener_internal_jb_entitle_now (void * jbd_call, guint s
 
   return jbd_call_func (service_port, 1, pid);
 }
+
+guint
+_frida_unc0ver_policy_softener_internal_connect (void)
+{
+  mach_port_t service_port = MACH_PORT_NULL;
+  kern_return_t kr;
+
+  kr = task_get_special_port (mach_task_self(), TASK_SEATBELT_PORT, &service_port);
+  if (kr != KERN_SUCCESS)
+    return MACH_PORT_NULL;
+
+  if (!MACH_PORT_VALID (service_port))
```

Comment:
Theoretically if this check fails because `service_port` is `MACH_PORT_DEAD`, will we be leaking a dead port right? (Can't remember the Mach semantics about those.)

Reply:
hmm i'm not sure this can really be `MACH_PORT_DEAD`, actually.

but maybe we could just ignore the check and let it fail (silently) later?

Reply:
Sounds good to me!